Migrating.md

```diff
@@ -51,6 +51,8 @@ params = { color };
 lil_gui.addColor( params, 'color' );
 ```
 
+_Note: lil-gui is automatically converting color spaces under the hood since THREE.Color provides getHex/setHex methods._
```

Comment:
this part of the Migrating section talks about how lil-gui allows you to forego the old `onChange` pattern from dat.gui. i'm thinking about changing this example to use a different lib with 0-1 RGB values (maybe PIXI?). this way we don't have to gloss over colorspaces.